beacon.go

```diff
@@ -17,16 +17,13 @@
 package beacon
```

Comment:
This should live in a separate package.  We can do that in another CL if you like, but it would be good to at least make the Query type have an exported API now so that's easy to do.